ethdb/privateapi/ethbackend.go

```diff
@@ -270,6 +291,74 @@ func (s *EthBackendServer) EngineGetPayloadV1(ctx context.Context, req *remote.E
 	return nil, fmt.Errorf("unknown payload")
 }
 
+// EngineGetPayloadV1, retrieves previously assembled payload (Validators only)
```

Comment:
The comment doesn't match the function

Reply:
it does, getPayload returns a previously assembled aka. proposed block

Reply:
oh wait, my bad